dvc.yaml

```diff
@@ -18,19 +25,7 @@ stages:
     - data/graph/
   train:
     cmd:
-    - sbatch -W machines/slurm.train-neural-lam.sh --config_path ./data/config.yaml --logger mlflow --logger_project neural_lam
-    params:
-    - data/training_params.yaml:
-      - model
-      - graph
-      - epochs
-      - batch_size
-      - hidden_dim
-      - processor_layers
-      - metrics_watch
-      - num_workers
-      - num_nodes
-      - val_steps_to_log
+    - sbatch -W ${slurm-defaults} machines/slurm.train.sh --config_path ./data/config.yaml --logger ${logger} --logger_project ${experiment} ${baseline-defaults}
```

Comment:
this should be `machines/slurm.train-neural-lam.sh` no?

Reply:
good catch!